packages/ui-extensions/src/types/Heading.ts

```diff
@@ -0,0 +1,20 @@
+type Level = 1 | 2 | 3;
```

Comment:
We should export this type. In checkout we will want to redeclare the `level` prop, even though it is already on the shared type, so that we can add the additional documentation that is relevant for checkout's automatic heading level (https://github.com/Shopify/argo/blob/main/packages/argo-checkout/src/components/Heading/Heading.ts#L12-L18)